src/main/webapp/query-control.js

```diff
@@ -137,9 +145,34 @@ function getHistory() {
         if (historyElement === null) return;
         addHistoryToPage(historyElement, historyList);
       });
-      new Splide('.splide', {
+      new Splide('#splide1', {
+        gap: 20,
+        fixedWidth: 230,
+        padding: 20,
+        pagination: false,
+      }).mount();
+    });
+}
+
+function getRecommendations() {
+  resetRecommendationList();
+  const recommendationList = document.getElementById('recommendation-list');
+  const fetchUrl = '/recommendations?user-id=' + getUserId();
+  fetch(fetchUrl).then(function(response) {
+    if (!response.ok) {
+      return;
+    } else {
+      return response.json();
+    }
+  })
+    .then(function(jsonResponse) {
+      // iterate through list of recommendations
+      jsonResponse.forEach((historyElement) => {
+        if (historyElement === null) return;
+        addHistoryToPage(historyElement, recommendationList);
+      });
+      new Splide('#splide2', {
```

Comment:
I kind of wish there was a way to tell it to only show arrows if there are more items that aren't visible on the page. My screen can fit up to 6 recommendations, so I never need the arrows, I can always see them all. But if someone else is on phone, they might still want them. IDK if this is possible, though.

Reply:
Could check window width right above this (window.innerWidth) and turn off arrows past a certain width (by adding "arrows: false" to the splide options)

Reply:
Fixed so that the user should only see arrows if their window is less than 1250 px